kong/plugins/proxy-cache/schema.lua

```diff
@@ -72,6 +72,7 @@ return {
             elements = { type = "string" },
           }},
           { vary_headers = { description = "Relevant headers considered for the cache key. If undefined, none of the headers are taken into consideration.", type = "array",
+            default = { "Accept-Encoding" },
```

Comment:
It seems that have introduced breaking change. I am not sure this is safe for our old version user.

Reply:
cc @dndx Can you help us to check it?

Reply:
I don't consider fixing a broken behavior a breaking change. ping @dndx